src/gui/windows/debug.zig

```diff
@@ -25,7 +25,7 @@ pub var window = GuiWindow {
 pub fn render() void {
 	draw.setColor(0xffffffff);
 	var y: f32 = 0;
-	draw.print("    fps: {d:.0} Hz{s}", .{1.0/main.lastDeltaTime.load(.monotonic), if(main.settings.vsync) @as([]const u8, " (vsync)") else ""}, 0, y, 8, .left);
+	draw.print("    fps: {d:.0} Hz{s}{s}", .{1.0/main.lastDeltaTime.load(.monotonic), if(main.settings.vsync) @as([]const u8, " (vsync)") else "", if(main.settings.fpsCap == null) " (unlimited)" else std.fmt.allocPrint(main.stackAllocator.allocator, " (limit: {d:.0} Hz)", .{main.settings.fpsCap.?}) catch unreachable}, 0, y, 8, .left);
```

Comment:
I would remove the unlimited tag. `(vsync) (unlimited)` also seems a bit contradicting.

Reply:
Also I think this is leaking memory.

Reply:
So I suppose the 3 cases should be these:
```
(vsync)
(limit: N) (vsync)
(unlimited)
```

Reply:
I'm would be fine with not having anything if it's unlimited.